subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -948,8 +948,7 @@ int lwm2m_init_message(struct lwm2m_message *msg)
 	}
```

Comment:
Please change the commit message so that the "fixes" line is not part of the description. Usually we have first commit description, followed an empty line, then the "Fixes #xxxxx" line, followed an empty line and then the signed-off.

Do we really need first to introduce some change, and then removing that change in the next commit. Looks like we could just squash these changes together.

Reply:
@jukkar the first commit's scope is bigger. It introduces an improvement to both message id as well as token generation. The second commit is the actual fix for the message id generation only. Thus squashing them together does not make sense. 

The first commit is also available as its separate pull request #28281. if you prefer to separate the concerns, I'm open to rebase this pull request should you decide to merge the other first.

Reply:
No need to separate the commits, it was just looking a bit unnecessary to first introduce a change, and then next commit would change the previous commit again.